src/strategies/MangroveOrder.sol

```diff
@@ -11,39 +11,26 @@
 
-pragma abicoder v2;
-
-import {IMangrove} from "src/IMangrove.sol";
-import {Forwarder, MangroveOffer} from "src/strategies/offer_forwarder/abstract/Forwarder.sol";
-import {IOrderLogic} from "src/strategies/interfaces/IOrderLogic.sol";
-import {SimpleRouter} from "src/strategies/routers/SimpleRouter.sol";
-import {TransferLib} from "src/strategies/utils/TransferLib.sol";
-import {MgvLib, IERC20} from "src/MgvLib.sol";
+import {IMangrove} from "mgv_src/IMangrove.sol";
+import {Forwarder, MangroveOffer} from "mgv_src/strategies/offer_forwarder/abstract/Forwarder.sol";
+import {IOrderLogic} from "mgv_src/strategies/interfaces/IOrderLogic.sol";
+import {SimpleRouter} from "mgv_src/strategies/routers/SimpleRouter.sol";
+import {TransferLib} from "mgv_src/strategies/utils/TransferLib.sol";
+import {MgvLib, IERC20} from "mgv_src/MgvLib.sol";
 
 ///@title MangroveOrder. A periphery contract to Mangrove protocol that implements "Good till cancelled" (GTC) orders as well as "Fill or kill" (FOK) orders.
-///@notice A GTC order is a market buy (sell) order complemented by a bid (ask) order, called a resting order, that occurs when the buy (sell) order was partially filled.
-/// * If the GTC is for some amount $a_goal$ at a price $p$, and the corresponding market order was partially filled for $a_now < a_goal$ at a price $p_synch <= p$,
-/// the resting order should be posted for an amount $a_later$ at price $p_async$ such that:
-//// $$ (a_now * p_synch + a_later * p_async) = a_goal * p $$ (modulo rounding error).
-/// * If the taker tolerates some slippage, i.e., $p=p_0+slippage$, the price of the resting order is computed considering $p_0$.
-/// * The resting order comes with a "time to live" after which the corresponding offer is no longer valid.
-/// * The resting order can be cancelled or updated by its owner at any time.
-/// E.g., `msg.sender` wishes to buy 1 ETH at a limit average price of 1500 USD/ETH + 1% slippage tolerance. After a market buy order:
-/// 1. `msg.sender` gets a partial fill of 0.582 ETH (0.6 ETH - 3% fee) for 850 USD (thus at a price of $850/0.6 =~ 1417$ USD/ETH).
-/// 2. This contract will then post a resting bid of 0.4 ETH for 650 USD (thus at a price of 1625 USD)
-/// 3. If the resting order is fully taken, the final price for `msg.sender` will indeed be 1500 USD/ETH.
-/// A FOK order is simply a buy or sell order that is either completely filled or cancelled. No resting order is posted.
+///@notice A GTC order is a buy (sell) limit order complemented by a bid (ask) limit order, called a resting order, that occurs when the buy (sell) order was partially filled.
+/// If the GTC is for some amount $a_goal$ at a price $p$, and the corresponding limit order was partially filled for $a_now < a_goal$,
+/// the resting order should be posted for an amount $a_later = a_goal - a_now$ at price $p$.
+///@notice A FOK order is simply a buy or sell limit order that is either completely filled or cancelled. No resting order is posted.
```

Comment:
See 5.5
This comment updates the simplified spec of the resting order price policy.